### ISSUE TITLE
[CI] Failure in {p0=search.aggregation/270_median_absolute_deviation_metric/missing value}

### ISSUE THREAD

**Issue by original-brownbear** [elastic/elasticsearch#64568]
```
./gradlew ':rest-api-spec:yamlRestTest' --tests "org.elasticsearch.test.rest.ClientYamlTestSuiteIT" -Dtests.method="test {p0=search.aggregation/270_median_absolute_deviation_metric/missing value}" -Dtests.seed=3E8CE2CF34AEA865 -Dtests.security.manager=true -Dtests.locale=en-ZA -Dtests.timezone=Africa/Lagos -Druntime.java=11
```

failed on master in https://gradle-enterprise.elastic.co/s/kyjhvgp6h7euc/console-log/raw

with a hanging client situation and subsequent timeout:

The interesting hanging is probably this one:

```
  2> "TEST-ClientYamlTestSuiteIT.test-seed#[3E8CE2CF34AEA865]" ID=18 WAITING on org.apache.http.concurrent.BasicFuture@549be88a
  2> 	at java.base@11.0.2/java.lang.Object.wait(Native Method)
  2> 	- waiting on org.apache.http.concurrent.BasicFuture@549be88a
  2> 	at java.base@11.0.2/java.lang.Object.wait(Object.java:328)
  2> 	at app//org.apache.http.concurrent.BasicFuture.get(BasicFuture.java:82)
  2> 	at app//org.apache.http.impl.nio.client.FutureWrapper.get(FutureWrapper.java:70)
  2> 	at app//org.elasticsearch.client.RestClient.performRequest(RestClient.java:279)
  2> 	at app//org.elasticsearch.client.RestClient.performRequest(RestClient.java:270)
  2> 	at app//org.elasticsearch.test.rest.ESRestTestCase.wipeAllIndices(ESRestTestCase.java:665)
  2> 	at app//org.elasticsearch.test.rest.ESRestTestCase.wipeCluster(ESRestTestCase.java:561)
  2> 	at app//org.elasticsearch.test.rest.ESRestTestCase.cleanUpCluster(ESRestTestCase.java:314)
  2> 	at jdk.internal.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
  2> 	at java.base@11.0.2/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  2> 	at java.base@11.0.2/java.lang.reflect.Method.invoke(Method.java:566)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:1004)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
  2> 	at app//org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  2> 	at app//org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
  2> 	at app//org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  2> 	at app//org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  2> 	at app//com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  2> 	at app//org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  2> 	at app//org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  2> 	at app//org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  2> 	at app//org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
  2> 	at app//com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
  2> 	at app//com.carrotsearch.randomizedtesting.ThreadLeakControl$$Lambda$202/0x00000008002ec840.run(Unknown Source)
  2> 	at java.base@11.0.2/java.lang.Thread.run(Thread.java:834)

```

Does not reproduce locally (tried it on OSX just like the test failure). Neither with the individual test nor when running the whole suite with that index. Marking this as core/infra since it seems we're hanging on deleting indices in general and not necessarily related to this specific test.

**Comment by mark-vieira** [comment]
Closing this as a duplicate of #58286.
